### ISSUE TITLE
Accessibility icon and flag does not have pointer

### ISSUE THREAD

**Issue by cemalokten** [yalla-coop/cost-of-living-support#133]
- Priority-4 (Low e.g. Content is not perfectly aligned)

**Describe the bug**
Accessibility icon and flag need pointer

**Team - do not edit**
@thejoefriel
